### PR TITLE
Debian: Fix fax numbers supplied via GTK print dialog

### PR DESCRIPTION
**Pull request by OdyX**
… removing a "Custom." prefix; do not choke if the GTK dialog sends "None" as phone number or pre-dial prefix

### CODE REVIEW

**Review by michaelrsweet — CHANGES_REQUESTED** [review]
Coding style doesn't match CONTRIBUTING.md - please update then I'll merge. Thanks!

**Comment by OdyX** [comment]
That's @tillkamppeter 's code. Would you be interested to fix this?

File: backend/ipp.c [hunk 1]
```diff
@@ -2825,7 +2825,19 @@ new_request(
...
 
         _httpDecodeURI(phone, keyword, sizeof(phone));
-        for (ptr = phone; *ptr;)
+	ptr = phone;
+
+	/* Weed out "Custom." in the beginning, this allows to put the
+	   "phone" option as custom string option into the PPD so that
+	   print dialogs not supporting fax display the option and
+	   allow entering the phone number. Print dialogs also send "None"
+	   if no phone number got entered, filter this, too. */
```
Comment (tillkamppeter) [inline backend/ipp.c:2834]: @OdyX, for me it seems that only this comment does not fulfill the coding conventions of `DEVELOPING.md` of CUPS (not `CONTRIBUTING.md`). Could you correct it, as you have already created the branch for this PR. It should be of the style
```
/*
 * ...
 * ...
 */
```
@michaelrsweet, am I correct? I do not see anything else which is wrong.

Comment (michaelrsweet) [reply backend/ipp.c:2834]: @tillkamppeter The rest of the code is line-wrapped (original code no longer does so - I gave up on keeping lines under 80 columns several years ago) and I'm not sure about the indentation.

Comment (tillkamppeter) [reply backend/ipp.c:2834]: So this means that each function call has to be a single line? No newline between `(` and `)`? @OdyX could you do this in your branch?

Comment (michaelrsweet) [reply backend/ipp.c:2834]: @tillkamppeter I mean that when you have a function call with arguments, you just continue the arguments on that line and let your editor do the wrapping for you...  For example:

```
httpAssembleURI(HTTP_URI_CODING_ALL, tel_uri, sizeof(tel_uri),
                "tel", NULL, NULL, 0, phone);
```

should be:

```
httpAssembleURI(HTTP_URI_CODING_ALL, tel_uri, sizeof(tel_uri), "tel", NULL, NULL, 0, phone);
```

Comment (tillkamppeter) [reply backend/ipp.c]: I have found a way to fix it in this PR by editing right in the browser, seems to be some admin functionality of GitHub.
I have now unbroken the function calls to be in one line and re-formatted the long comment.

**Comment by OdyX** [comment]
OK. I have (force-)pushed a squash of Till's changes, with whitespace fixup; let's not add tabs-mixed-with-spaces where we can.

**Review by michaelrsweet — APPROVED** [review]

**Pull request merged**